src/main/java/roomescape/application/service/TossPaymentService.java

```diff
@@ -0,0 +1,57 @@
+package roomescape.application.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+import roomescape.application.support.TossPaymentWithHttpClient;
+import roomescape.common.exception.PaymentClientException;
+import roomescape.dto.request.TossPaymentConfirmDto;
+import roomescape.dto.request.TossPaymentRequestDto;
+import roomescape.dto.response.TossPaymentConfirmResponseDto;
+import roomescape.model.PaymentTargetType;
+import roomescape.model.ReservationTicket;
+import roomescape.model.TossPayment;
+import roomescape.persistence.repository.TossPaymentRepository;
+
+@Service
+@RequiredArgsConstructor
+public class TossPaymentService {
+
+    private final TossPaymentRepository tossPaymentRepository;
+    private final TossPaymentWithHttpClient tossPaymentWithHttpClient;
+
+    @Retryable(
+            maxAttempts = 2,
+            backoff = @Backoff(delay = 1_000)
+    )
+    public void processPayment(TossPaymentRequestDto tossPaymentRequestDto,
+                               String requestKey) {
+        TossPaymentConfirmDto tossPaymentConfirmDto = new TossPaymentConfirmDto(
+                tossPaymentRequestDto.paymentKey(),
+                tossPaymentRequestDto.orderId(),
+                tossPaymentRequestDto.amount()
+        );
```

Comment:
이 메서드를 호출할때 변경해서 매개변수로 받는게 올바른 메서드의 시그니처가 아닐까요?!
```suggestion
    public void processPayment(TossPaymentConfirmDto tossPaymentConfirmDto,
                               String requestKey) {
        
```